general/clients/installing-kodi.md

```diff
@@ -0,0 +1,54 @@
+---
+uid: clients-installing-kodi
+title: Installing Kodi
+---
+#Installing Kodi
+
+## Installation Process
+
+1. Download the repository installer [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip)
+    * It will be saved as `repository.jellyfin.kodi.zip`
+2. Install the Jellyfin repository
+    * Open Kodi and navigate to "Add-on Browser"
+    * Select "Install from zip file"
+        * If prompted, enter settings and enable "Unknown Sources", then go back
+    * Select the newly downloaded file and it will be installed
+3. Install Jellyfin for Kodi
+    * From within Kodi, navigate back to "Add-on Browser"
+    * Select "Install from repository"
+    * Choose "Kodi Jellyfin Addons"
+    * "Video Add-ons"
+    * Select the Jellyfin add-on and choose install
+4. Within a few seconds you should be prompted for your server-details.
+    * If a Jellyfin server is detected on your local network, it will displayed in the popup
+    * If a Jellyfin server is not detected on your local network, select "Manually add server"
+        * Note that if you have a baseurl set, you should append that value to the end of the host field.  ie: `192.168.0.10:8096/jellyfin`
```

Comment:
```suggestion
        * Note that if you have a baseurl set, you should append that value to the end of the host field.  ie: `192.168.0.10:8096`
```

Reply:
Isn't that correct? I changed it to your suggestion, but on second thought, I think the original was correct.

Reply:
Yeah, the default is now a blank string so this is correct.